MapboxNavigation/NavigationViewController.swift

```diff
@@ -450,6 +455,14 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
         }
     }
     
+    func ETAIsStale() {
```

Comment:
This sounds like the name of a property that returns a value without side-effects, which isn’t the case. How about `updateETA()`?